web/cypress/integration/route/online-debug.spec.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-undef */
+
+context('Online debug', () => {
+  const ServerHost = '139.217.190.60';
```

Comment:
why not using `http://localhost:9000`?

Reply:
> why not using `http://localhost:9000`?

Actually it would be better to use APISIX's address here, and I'm not sure this if this address is also for APISIX(It shouldn't be, is it)

BTW, will github CI work well if  I use `http://localhost:9000`? @LiteSun

Reply:
Not sure if APISIX is started on the backend.  Need @nic-chen to confirm.

What I am trying to say is: we should not rely on outside services for testing 😊

Reply:
> Not sure if APISIX is started on the backend. Need @nic-chen to confirm.
> 
> What I am trying to say is: we should not rely on outside services for testing 😊

agree with you, we should not rely on outside services for testing. if needed, we could run it in CI.